src/python/pants/core/goals/test_test.py

```diff
@@ -248,8 +252,8 @@ def test_summary(rule_runner: RuleRunner) -> None:
     assert stderr == dedent(
         """\
 
-        ✓ //:good succeeded.
-        𐄂 //:bad failed.
+        ✓ //:good succeeded in 999 ms (memoized).
+        𐄂 //:bad failed in 999 ms (memoized).
```

Comment:
I'm not sure how to go about mocking a different RunId for the tests so that I can cover the non-memoized cases.